### ISSUE TITLE
[WPF] Radio buttons' border in "Event settings" have insufficient contrast against its adjacent color

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#865]
The following AI-windows mode(s) have this issue:
- High Contrast White mode

**Screenshots or .GIF**
#### Screenshot of the issue in High Contrast White mode
![12 1](https://user-images.githubusercontent.com/22068434/88355438-69230280-cd19-11ea-9eba-e65292946276.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by juchampa** [comment]
This is a WPF specific color contrast issue
